keyboards/keebio/iris/keymaps/blucky/keymap.c

```diff
@@ -0,0 +1,88 @@
+#include QMK_KEYBOARD_H
+
+extern keymap_config_t keymap_config;
+
+#define _QWERTY 0
+#define _LAYER1 1
+#define _LAYER2 2
+#define _LAYER3 3
+
+#define WM_0     LGUI(KC_ENT)
+#define WM_1     LGUI(KC_BSPC)
+#define WM_2     LGUI(KC_DEL)
+#define WM_3     SGUI(KC_ENT)
+#define WM_4     SGUI(KC_BSPC)
+#define WM_5     SGUI(KC_DEL)
+#define WM_6     SGUI(KC_TAB)
+
+#define LAYER1   TT(_LAYER1)
+#define LAYER2   TT(_LAYER2)
+
+#define CS_U     C(S(KC_UP))
+#define CS_D     C(S(KC_DOWN))
+#define CS_SPC   C(S(KC_SPC))
+
+#define NONE     KC_NO
```

Comment:
QMK defines an alias for this keycode automatically.

https://github.com/qmk/qmk_firmware/blob/48cc61b18883a482ad8494e005054e64be5cd8bf/quantum/quantum_keycodes.h#L31

Reply:
I made my own definition because it used less characters than both that already existed

Reply:
So the rational behind is _______ and XXXXXXX is that automated tools can find files and change them automatically. That's not really possible with custom defines. We'd appreciate it if you'd change them back place.

Reply:
updated